src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -42,6 +42,8 @@ internal class AD7DebugSession : DebugAdapterBase, IDebugPortNotify2, IDebugEven
 
         private readonly DebugEventLogger m_logger;
         private readonly Dictionary<string, Dictionary<int, IDebugPendingBreakpoint2>> m_breakpoints;
+        private readonly List<IDebugCodeContext2> m_gotoCodeContexts = new List<IDebugCodeContext2>();
```

Comment:
>List [](start = 25, length = 4)

I would suggest using `HandleCollection` instead. HandleCollection should really internally use a List instead of a Dictionary, but what you have here is exactly what HandleCollection want meant to do.